csm_web/csm_web/settings.py

```diff
@@ -163,6 +162,18 @@
 
 USE_TZ = True
 
+# Media files and S3 configuration
+# https://docs.djangoproject.com/en/3.0/topics/files/
+MEDIA_URL = "/media/"
+MEDIA_ROOT = os.path.join(BASE_DIR, "media")
+
+# if DJANGO_ENV == PRODUCTION:
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+AWS_STORAGE_BUCKET_NAME = os.environ.get("AWS_STORAGE_BUCKET_NAME")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
+AWS_STORAGE_BUCKET_NAME = os.environ.get("AWS_STORAGE_BUCKET_NAME")
+# Defaults to public_read, which seems to prevent us from uploading to the bucket
```

Comment:
Setting Access Control List to `None` scares me, we should definitely look into this more before deploying anything like this.